recipes/pytorch-frame/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "0.2.5" %}
+
+package:
+  name: pytorch-frame
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/p/pytorch-frame/pytorch_frame-{{ version }}.tar.gz
+  sha256: 927a322a1c8cd4af386e124aa0932f2a0aba415c8a1f593c2ed605e08a9f65d5
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - flit-core >=3.2,<4
+    - pip
+  run:
+    - python >={{ python_min }}
+    - numpy
+    - pandas
+    - pytorch
+    - tqdm
+    - pyarrow
+    - pillow
+
+test:
+  imports:
+    - torch_frame
+    - torch_frame.config
+    - torch_frame.data
+    - torch_frame.datasets
+    - torch_frame.gbdt
+    - torch_frame.nn
+    - torch_frame.transforms
+    - torch_frame.utils
+  commands:
+    - pip check
+  requires:
+    - python {{ python_min }}
+    - pytorch-cpu >=2.5.1  # first conda-forge version for all platforms
```

Comment:
Will this be needed to run as well? Why only on test?

Reply:
I think it's useful to control which pytorch variant is installed for test. I don't know if it's strictly necessary.

Reply:
I'm afraid that, it will pass with this version but not the others.